test/unit/server/soroban/get_events_test.js

```diff
@@ -239,7 +239,7 @@ let getEventsResponseFixture = [
     type: "system",
     ledger: "1",
     ledgerClosedAt: "2022-11-16T16:10:41Z",
-    contractId,
+    contractId: '',
```

Comment:
ideally, would have been good to see this trigger a test failure, it may be worth adding a new test case to assert on new expectation that `contractId` in parsed event should be absent now if raw rpc event was empty correct?:
```
  it("can parse empty contractIds ", ...)
```

Reply:
Without the code change, this test did in fact fail exactly as described in the Discord issue! so I think this is implicitly covered. But I can add an explicit test if you feel it is clearer